src/core/sys/linux/fcntl.d

```diff
@@ -17,6 +17,24 @@ enum {
     FALLOC_FL_UNSHARE_RANGE = 0x40
 }
 
+// From asm-generic/fcntl.h
+/*
```

Comment:
Any reason why you didn't copy the title as well: "Open File Description Locks"?